backend/util/util.go

```
@@ -20,3 +24,14 @@ func Enumerate[T any](cookieIndex *int, iterator iter.Seq[T]) iter.Seq2[int, T]
 		}
 	}
 }
+
+func GetLocalIP() string {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
```

golangci-lint (2.12.2)

[error] net.Dial must not be called. use (*net.Dialer).DialContext

(noctx)